src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" action="https://www.google.com/">
+      <div class="search-bar-big">
+        <label for="icon_key_big">
+          <img
+             class="icon_big" 
+             src="images/Search.svg" 
+             alt="icon_big"
+          >
+          <input id="icon_key_big" data-qa="keypress"
+          class="input input_big" type="text" 
+          placeholder="Try &quot;Los Angeles&quot;"> 
+        </label>
+      </div>
+    </form>
+    <form data-qa="small" action="https://www.google.com/">  
+      <div class="search-bar-small">  
+        <label for="icon_key_small">
+          <img
+             class="icon_small" 
+             src="images/Search.svg" 
+             alt="icon_small">
+          <input id="icon_key_small" data-qa="keypress"
+          class="input input_small" type="text"
+          placeholder="Try &quot;Los Angeles&quot;"> 
```

Comment:
```suggestion
          <input
            id="icon_key_small"
            data-qa="keypress"
            class="input input_small" 
            type="text"
            placeholder="Try &quot;Los Angeles&quot;"
          > 
```
all in one line or every attribute from new line